logos/logos-workernode/tests/test_logos_bridge.py

```
@@ -165,6 +165,179 @@ async def post(self, url, headers=None, json=None):  # noqa: ARG002
+        decrement_active_requests=AsyncMock(return_value=None),
+    )
+    app.state.lane_manager = lane_manager
+
+    client = LogosBridgeClient(
+        app,
+        LogosConfig(enabled=True, logos_url="https://logos.example", shared_key="secret"),
+    )
+
+    class _Resp:
+        status_code = 200
+        headers = {"content-type": "text/plain; charset=utf-8"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)